src/tools/build-manifest/src/main.rs

```diff
@@ -544,7 +544,13 @@ impl Builder {
                 if target_name.contains(substr) {
                     let t = Target::from_compressed_tar(self, &tarball_name!(fallback_target));
                     // Fallbacks must always be available.
```

Comment:
I think this comment here needs adjusting, since it is no longer true that fallbacks must be available.